compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -256,7 +256,7 @@ class CompilationTests extends ParallelTesting {
       // as well as bootstrapped compiler:
       defaultOutputDir + dotty1Group + "/dotty/:" +
       Jars.dottyInterfaces,
-      Array("-Ycheck-reentrant")
+      Array("-Ycheck-reentrant", "-Yemit-tasty-in-class")
```

Comment:
Why not the default?

Reply:
It triggers an idempotency test failure. I have partially fixed it but still working on finding the second source of non-determinism.